packages/status-js/vite.config.ts

```diff
@@ -4,12 +4,28 @@ import { defineConfig } from 'vite'
 
 import { dependencies } from './package.json'
 
+import type { Alias } from 'vite'
+
+const isTest = process.env.NODE_ENV === 'test'
```

Comment:
I would do it the same, but I remember reading this from their [docs](https://vitest.dev/guide/#configuring-vitest):

> Use process.env.VITEST or mode property on defineConfig (will be set to test if not overridden) to conditionally apply different configuration in vite.config.ts

Reply:
Thx 👍, done.